pkg/providers/tinkerbell/pbnj/client.go

```diff
@@ -0,0 +1,60 @@
+package pbnj
```

Comment:
Correct me if im wrong, but didn't we decide to keep it under `pkg/clients/pbnj`?

Reply:
I am fine either way.
But i think we decided this might be specific to tinkerbell provider and not a more generic "client" and hence we wanted to keep it in providers package?

Reply:
yeah, I am also fine either way but I thought we decided to put it under Tinkerbell provider considering it is only for Tinkerbell.

Reply:
The `client` package is a bit of an organizational thing and not strictly necessary. The `pbnj` package provides constructs and functions related to pbnj within the context of tinkerbell so its totally appropriate to have it here.